layouts/partials/head.html

```diff
@@ -59,36 +63,25 @@
         }))
   </script>
   {{if $.Site.Data.config.enableSPA}}
+  {{ $router := resources.Get "js/router.js" | resources.Fingerprint "md5" |
+  resources.Minify }}
   <script type="module">
-    import { router, navigate } from "https://unpkg.com/million/dist/router.mjs";
-    // SPA navigation for access later
-    window.navigate = navigate;
-    // We only mutate document.title and content within .singlePage element
-    router(".singlePage");
-    const callback = () => {
-      // requestAnimationFrame() delays graph draw until SPA routing is finished
-      const draw = () => {
-        const container = document.getElementById("graph-container");
-        // retry if the graph is not ready
-        if (!container) return requestAnimationFrame(draw);
-        // clear the graph in case there is anything within it
-        container.textContent = "";
+    import { init } from '{{$router.Permalink}}';
+    init(() => {
+      drawGraph(
+        {{strings.TrimRight "/" .Page.Permalink}},
```

Comment:
As mentioned in #122 , can remove this as `_url` is no longer in use

Reply:
I think this parameter should be kept in this PR as it depends on #122. My thought process is that once it merges the changes in this PR can be overwritten by #122. Otherwise, we would just be reimplementing #122